test/controllers/AuthorizedToViewControllerSpec.scala

```diff
@@ -371,75 +416,63 @@ class AuthorizedToViewControllerSpec extends SpecBase with ShouldMatchers {
 
         when(mockDataStoreService.getXiEori(any)).thenReturn(Future.successful(Option("XI123456789")))
 
-        running(app) {
-          val request = fakeRequest(POST, routes.AuthorizedToViewController.onSubmit().url)
-            .withFormUrlEncodedBody("value" -> "GB123456789012")
-          val result  = route(app, request).value
-          val html    = Jsoup.parse(contentAsString(result))
-
-          status(result)                                            shouldBe OK
-          html.text().contains("Search results for GB123456789012") shouldBe true
+        validateRedirectToOnSearchAndThen(gbEORI, app) { (result, html) =>
+          status(result)                                      shouldBe OK
+          html.text().contains(s"Search results for $gbEORI") shouldBe true
         }
       }
 
-    "return OK if there are no authorities returned for both GB/XI EORI for a account and" +
-      " display no authorities page" in new Setup {
+    "return SEE_OTHER if there are no authorities returned for both GB/XI EORI for a account and" +
```

Comment:
we hav ea space at the start and end of the lines 426-429 so it'll be double spaced in the print out?